tests/manager/services/container_registry/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from ai.backend.manager.services.container_registry.processors import ContainerRegistryProcessors
+from ai.backend.manager.services.container_registry.service import ContainerRegistryService
+
+
+@pytest.fixture
+def processors(extra_fixtures, database_fixture, database_engine):
+    container_registry_service = ContainerRegistryService(
+        db=database_engine,
+    )
+    return ContainerRegistryProcessors(container_registry_service, [])
```

Comment:
rename to `container_registry_processors`

Reply:
Since it's the processors defined in `container_registry/conftest.py`, wouldn't it be fine to just leave it as processors?

Reply:
In a situation where there may be multiple processors, it seems like a good idea to reduce the room for error.

Reply:
I agree with you. But this file has only one processors.